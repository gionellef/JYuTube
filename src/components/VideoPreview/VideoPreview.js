import React from 'react';
import { Image } from 'semantic-ui-react';

import './VideoPreview.scss';

export class VideoPreview extends React.Component {
  render () {
    return (
      <div className='video-preview'>
        <div className='image-container'>
          <Image src='http://via.placeholder.com/210x118' />
          <div className='time-label'>
            <span>5:22</span>
          </div>
        </div>
      </div>
    );
  }
}