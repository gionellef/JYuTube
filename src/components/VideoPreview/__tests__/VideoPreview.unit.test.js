import React from 'react';

import { shallow } from 'enzyme';
import { VideoPreview } from '../VideoPreview';


describe('VideoPreview', () => {
  test('renders VideoPreview vertical / default', () => {
    const wrapper=shallow(<VideoPreview />);
    expect(wrapper).toMatchSnapshot();
  });

  test('renders VideoPreview horizontal', () => {
    const wrapper = shallow(<VideoPreview hoizontal={true}/>);
    expect(wrapper).toMatchSnapshot();
  });

});