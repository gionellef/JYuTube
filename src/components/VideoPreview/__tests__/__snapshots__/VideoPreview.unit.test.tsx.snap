// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPreview renders VideoPreview horizontal 1`] = `<div />`;

exports[`VideoPreview renders VideoPreview vertical / default 1`] = `<div />`;
